pkg_building.es.Rmd

```diff
@@ -1,513 +1,511 @@
+donde issue\_id es el número de la cuestión en el repositorio de revisión de software. Por ejemplo, la insignia de [`rtimicropem`](https://github.com/ropensci/rtimicropem) revisión utiliza el número 126, ya que es el número de [número de edición de revisión](https://github.com/ropensci/software-review/issues/126). La insignia indicará primero "en revisión" y luego "revisado" una vez que tu paquete haya sido incorporado (cuestión etiquetada como "aprobada" y cerrada), y enlazará con la cuestión de revisión.
 
-* If your README has many badges consider ordering them in an html table to make it easier for newcomers to gather information at a glance. See examples in [`drake` repo](https://github.com/ropensci/drake) and in [`qualtRics` repo](https://github.com/ropensci/qualtRics/). Possible sections are 
-    * Development (CI statuses cf [CI chapter](#ci), Slack channel for discussion, repostatus)
-    * Release/Published ([CRAN version and release date badges from METACRAN](https://www.r-pkg.org/services#badges), [CRAN checks API badge](https://github.com/ropensci/cchecksapi#badges), Zenodo badge)
-    * Stats/Usage (downloads e.g. [download badges from METACRAN](https://www.r-pkg.org/services#badges))
-  The table should be more wide than it is long in order to mask the rest of the README.
+- Si tu README tiene muchas insignias, considera la posibilidad de ordenarlas en una tabla html para facilitar a los recién llegados la obtención de información de un vistazo. Consulta los ejemplos en [`drake` repo](https://github.com/ropensci/drake) y en [`qualtRics` repo](https://github.com/ropensci/qualtRics/). Las secciones posibles son
+  
+  - Desarrollo (estados de CI cf [capítulo CI](#ci)canal de Slack para la discusión, repostatus)
+  - Liberación/Publicación ([Insignias de la versión de CRAN y de la fecha de publicación de METACRAN](https://www.r-pkg.org/services#badges), [Insignia de la API de comprobación de CRAN](https://github.com/ropensci/cchecksapi#badges), insignia de Zenodo)
+  - Estadísticas/Uso (descargas, por ejemplo [insignias de descarga de METACRAN](https://www.r-pkg.org/services#badges))
+    La tabla debe ser más ancha que larga para enmascarar el resto del LÉAME.
 
-* If your package connects to a data source or online service, or wraps other software, consider that your package README may be the first point of entry for users.  It should provide enough information for users to understand the nature of the data, service, or software, and provide links to other relevant data and documentation.  For instance,
-a README should not merely read, "Provides access to GooberDB," but also include,
```

Comment:
This might be a language thing, but what is here goober supposed to mean? My guess is that this is a funny and silly fake example. Is this about [Goober and the Ghost Chasers](https://en.wikipedia.org/wiki/Goober_and_the_Ghost_Chasers)?